subprojects/docs/src/docs/userguide/migration/upgrading_version_7.adoc

```diff
@@ -117,6 +117,14 @@ tasks.withType<KotlinCompile>().configureEach {
 }
 ```
 
+==== Upgrade to Groovy 3.0.12
+
+Groovy has been updated to https://groovy-lang.org/releasenotes/groovy-3.0.html[Groovy 3.0.12].
+
+==== Upgrade to PMD 6.48.0
+
+PMD has been updated to https://pmd.github.io/latest/pmd_release_notes.html[PMD 6.48.0].
```

Comment:
Can we link to a version specific page instead? https://pmd.github.io/pmd-6.48.0/pmd_release_notes.html

Reply:
Yes, I couldn't find these pages before.